File: plugins/swift-export/testData/simple.golden.kt
import kotlin.native.internal.ExportedBridge

@ExportedBridge("namespace1_main_foobar")
public fun namespace1_main_foobar(param: Int): Int {
    val result = namespace1.main.foobar(param)
    return result
}

@ExportedBridge("namespace1_foo")
public fun namespace1_foo(): Int {
    val result = namespace1.foo()
    return result
}

@ExportedBridge("namespace2_bar")
public fun namespace2_bar(): Int {
    val result = namespace2.bar()
    return result
}

@ExportedBridge("__root___foo")
public fun __root___foo(): Int {
    val result = foo()
    return result
}

File: plugins/swift-export/testData/simple.golden.swift
enum namespace1 {
    enum main {
        public static func foobar(
            param: Swift.Int32
        ) -> Swift.Int32 {
            return namespace1_main_foobar(param)
        }
    }
    public static func foo() -> Swift.Int32 {
        return namespace1_foo()
    }
}

enum namespace2 {
    public static func bar() -> Swift.Int32 {
        return namespace2_bar()
    }
}

public func foo() -> Swift.Int32 {
    return __root___foo()
}
